ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java

```diff
@@ -0,0 +1,9 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {
+	
+	//code1
+	//code
```

Comment:
change the xpath
do not hardcode the date

Reply:
fixed